### PR TITLE
Bugfix/apparatus entries extraction

### PR DESCRIPTION
**Pull request by giuliac89**

### CODE REVIEW

File: src/app/models/evt-models.ts [hunk 1]
```diff
@@ -139,14 +139,12 @@ export class ApparatusEntry extends GenericElement {
     id: string;
     content: Reading[];
     notes: Note[];
-    variance: number;
```
[the remark this answers is not in the document]
Comment (giuliac89) [reply src/app/models/evt-models.ts:142]: We said to handle these computation in the component. I'm wrong?

File: src/app/models/evt-models.ts [hunk 2]
```diff
@@ -139,14 +139,12 @@ export class ApparatusEntry extends GenericElement {
...
     originalEncoding: string;
 }
 
 export class Reading extends GenericElement {
     id: string;
     witIDs: string[];
-    significant: boolean;
```
[the remark this answers is not in the document]
Comment (giuliac89) [reply src/app/models/evt-models.ts:149]: We said to handle these computation in the component. I'm wrong?

File: src/app/services/xml-parsers/app-parser.ts [hunk 3]
```diff
@@ -17,7 +15,7 @@ export class RdgParser extends EmptyParser implements Parser<XMLElement> {
             attributes: this.attributeParser.parse(rdg),
             witIDs: this.parseReadingWitnesses(rdg) || [],
             content: this.parseAppReadingContent(rdg),
-            significant: this.readingIsSignificant(rdg),
```
[the remark this answers is not in the document]
Comment (giuliac89) [reply src/app/services/xml-parsers/app-parser.ts:20]: We said to handle these computation in the component. I'm wrong?

File: src/app/services/xml-parsers/app-parser.ts [hunk 4]
```diff
@@ -106,7 +86,14 @@ export class AppParser extends EmptyParser implements Parser<XMLElement> {
     }
 
     private parseAppReadings(appEntry: XMLElement): Reading[] {
-        return Array.from(appEntry.querySelectorAll(this.readingTagName))
-            .map((rdg: XMLElement) => this.rdgParser.parse(rdg));
+        return Array.from(appEntry.querySelectorAll(`${this.readingTagName}, ${this.lemmaTagName}`))
+            .filter((el) => el.closest(this.appEntryTagName).getAttribute('xml:id') === appEntry.getAttribute('xml:id'))
```
[the remark this answers is not in the document]
Comment (giuliac89) [reply src/app/services/xml-parsers/app-parser.ts:90]: This way doesn’t work at all. What about if the <rdg> inside the nested apparatus is inside a <rdgGroup>?

[the remark this answers is not in the document]
Comment (ChiaraDipi) [reply src/app/services/xml-parsers/app-parser.ts:90]: Ok, then something like 
```
.filter((el) => el.closest('app') === appEntry)
```

File: src/app/models/evt-models.ts [hunk 5]
```diff
@@ -137,16 +137,15 @@ export interface WitnessGroup {
 }
 export class ApparatusEntry extends GenericElement {
     id: string;
-    content: Reading[];
+    lemma: Reading[];
```
Comment (szenzaro) [inline src/app/models/evt-models.ts:140]: Can the lemma have more than one reading?

File: src/app/services/xml-parsers/index.ts [hunk 6]
```diff
@@ -22,17 +23,18 @@ export const parseF: { [T in SupportedTagNames]: Parser<XMLElement> } = {
     glyph: createParser(GlyphParser, parse),
     graphic: createParser(GraphicParser, parse),
     lb: createParser(LBParser, parse),
+    lem: createParser(LemmaParser, parse),
```
Comment (szenzaro) [inline src/app/services/xml-parsers/index.ts:26]: In which case the `RdgParser` is different from the `LemmaParser`?

File: src/app/models/evt-models.ts (hunk 5 above)
Comment (ChiaraDipi) [reply src/app/models/evt-models.ts:140]: I think no, at least at first level

**Pull request merged**